### PR TITLE
Connecting chat

### PR DESCRIPTION
**Pull request by ConConRob**
# Description
Can now: 
- join a queued chat
- close a active chat
- send and receive live messages in a active chat

still need to make it so 
- queued tickets receive live messages
- clean things up and test

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
#29 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge

- [x] Complete, but not tested (may need new tests)

- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Manually tested

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [ ] I have commented my code, particularly in hard-to-understand areas

- [ ] I have made corresponding changes to the documentation, if relevant

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/Logout.js [hunk 1]
```diff
@@ -2,14 +2,24 @@ import React from 'react';
...
+  return <Redirect to='/' />;
 };
 
 Logout.propTypes = {
   logout: PropTypes.func.isRequired,
 };
 
-export default Logout;
\ No newline at end of file
+function mstp() {
```
Comment (Pav0l) [inline src/components/Logout.js]: ctrl+shift+f and replace `mstp` with `mapStateToProps` please 🙏

File: src/components/layout/navbar/NavBar.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ import { NavLink } from 'react-router-dom';
...
-    <nav className="nav-wrapper navy darken-2">
-      <div className="container">
-        <NavLink to="/" className="brand-logo">
-          <header className="App-header">
-            <img src={logo} className="App-logo" alt="logo" />{' '}
+    <nav className='nav-wrapper navy darken-2'>
+      <div className='container'>
+        <NavLink to='/' className='brand-logo'>
+          <header className='App-header'>
+            <img src={logo} className='App-logo' alt='logo' />{' '}
```
Comment (Pav0l) [inline src/components/layout/navbar/NavBar.js]: JSX syntax with double quotes

File: src/store/actions/chat.js [hunk 3]
```diff
@@ -56,6 +58,7 @@ export const addQueuedChat = chatLog => {
 };
 
 export const removeQueuedChat = chat_id => {
+  debugger;
```
Comment (Pav0l) [inline src/store/actions/chat.js]: 🙏 ❌ De 🐛 er

File: src/views/Billing.js [hunk 4]
```diff
@@ -127,7 +125,7 @@ Billing.propTypes = {
 
 const mapStateToProps = state => ({
   hotel: state.hotel,
-  hotel_id: state.currentUser.hotel_id,
+  hotel_id: state.currentUser.hotel_id, // do we need to bring this and the hotel object in
```
Comment (Pav0l) [inline src/views/Billing.js]: @mark-marshall will change this

File: src/views/CompanySettings.js [hunk 5]
```diff
@@ -22,64 +29,95 @@ class CompanySettings extends React.Component {
...
-      [name]: value
+      [name]: value,
     });
   };
 
   handleSubmit(hotelId, dispatchUpdateHotel) {
-
-    return (event) => {
+    return event => {
       console.log(event);
```
Comment (Pav0l) [inline src/views/CompanySettings.js]: 🙏 ❌ `console.log`

File: src/components/layout/navbar/NavBar.js (hunk 2 above)
Comment (ConConRob) [reply src/components/layout/navbar/NavBar.js]: I'm using prettier

**Pull request merged**